NOTES.WIN

```diff
@@ -35,24 +35,51 @@
  In addition to the requirements and instructions listed in INSTALL,
  these are required as well:
 
- - Perl. We recommend ActiveState Perl, available from
-   https://www.activestate.com/ActivePerl. Another viable alternative
-   appears to be Strawberry Perl, http://strawberryperl.com.
-   You also need the perl module Text::Template, available on CPAN.
+ - Perl.
+   We recommend Strawberry Perl, available from http://strawberryperl.com.
+   An alternative is ActiveState Perl, https://www.activestate.com/ActivePerl. 
+   You also need the perl modules Text::Template and Win32::Console,
```

Comment:
"You *may* also...", and as far as we know, that's only for newer versions of ActivePerl

Reply:
done